src/Controller/GameManager.js

```diff
@@ -0,0 +1,63 @@
+const OutputView = require('../View/OutputView');
+const InputView = require('../View/InputView');
+const ComputerAnswer = require('../Model/ComputerAnswer');
+const Validator = require('../Model/Validator');
+const Quit = require('../libs/Quit');
+const { MESSAGE, OPTION } = require('../libs/const');
+
+class GameManager {
+  #computerAnswer;
+
+  constructor() {
+    this.#computerAnswer = new ComputerAnswer();
+    this.#start();
+  }
+
+  #start() {
+    OutputView.printMessage(MESSAGE.start);
+    this.#play();
+  }
+
+  #play() {
+    InputView.readPlayerAnswer(this.#handlePlayerAnswer.bind(this));
+  }
+
+  #handlePlayerAnswer(playerAnswer) {
+    Validator.playerAnswer(playerAnswer);
+
+    OutputView.printResult(
+      this.#computerAnswer.comparePlayerAnswer(playerAnswer)
+    );
+
+    this.#actionAboutPlayerAnswer(playerAnswer);
+  }
+
+  #actionAboutPlayerAnswer(playerAnswer) {
+    if (this.#computerAnswer.isThreeStrike(playerAnswer)) this.#end();
+
+    this.#play();
+  }
+
+  #end() {
+    InputView.readCommand(this.#handleCommand.bind(this));
+  }
+
+  #handleCommand(option) {
+    Validator.playerOption(option);
+
+    this.#actionAboutCommand(option);
+  }
+
+  #actionAboutCommand(option) {
+    if (option === OPTION.restart) return this.#restart();
+
+    return Quit.game();
+  }
+
+  #restart() {
+    this.#computerAnswer.resetValue();
+    this.#play();
+  }
```

Comment:
앗 저는 computerAnswer를 Class로 만드신 이유가 재할당 때문이라고 생각했는데, 이렇게 resetValue()가 따로 있으면 객체에 좀 더 가까운 활용같다는 생각이 들어요! 아래와 같이 쓰시면 클래스를 좀 더 잘 활용하는 방법이 될 것 같습니다 👍
```suggestion
  #restart() {
    this.#computerAnswer = new ComputerAnswer(); // 인스턴스 재생성
    this.#play();
  }
```

Reply:
좋은 지적 감사합니다! 아직 객체, 클래스에 대한 명확한 구분이 어려워 헤매고 있는 중입니다ㅠㅠ😭 객체와 클래스, 언제 무엇을 사용하는지에 대한 기준을 찾아보며 공부를 해야겠어요ㅎㅎ